include/gadget/filesystem.h

```diff
@@ -6,7 +6,7 @@
 
 // clang-format off
 #define MAX_PERCPU_BUFSIZE (1 << 15)  // set by the kernel as an upper bound
-#define GADGET_PATH_MAX    512        // smaller than PATH_MAX to reduce memory consumption
+#define GADGET_PATH_MAX     4096
```

Comment:
This constant is also used for `struct event` in gadgets trace_exec, trace_mount, fdpass, fsnotify, top_file. So this is increasing the size of events in the ring buffer.

I think we need a different define for gadgets and for filesystem.h's internals.

Reply:
Good catch! You're right, the memory usage was being increased a lot:

```bash 
$ sudo gadgetctl run bpfstats
GADGE… GADGETNAME   GADGETIMAGE  PRO… PROGNAME     PROGTYPE            RUNTIME RUNCOUNT MAPMEM… MA… COMMS       PIDS
                    trace_exec      0                                      0ns        0   24 MB   7
```

After your suggestion it's back to the same size as main:

```bash 
$ sudo gadgetctl run bpfstats
GADGE… GADGETNAME   GADGETIMAGE  PRO… PROGNAME     PROGTYPE            RUNTIME RUNCOUNT MAPMEM… MA… COMMS       PIDS
                    trace_exec      0                                      0ns        0  9.3 MB   7
```